### PR TITLE
Document batch product search

### PR DESCRIPTION
**Pull request by janbuecker**

### CODE REVIEW

File: source/developers-guide/shopware-5-search-bundle/index.md [hunk 1]
```diff
@@ -231,3 +231,48 @@ In most cases, only a list of product numbers is not enough for the store front.
...
+
+## Optimized Batch Search
+
+As of Shopware 5.3, it is possible to search for multiple critera objects at once. In addition, the fetch process for products will be minimized by fetching a product number only once. An optimization service will then try to optimize the search request by combining identical criteria objects into one. So there will be less search requests for identical criterias.
+
+Another advantage is that you won't get the same results with identical searches.
+
+### Example: Emotion Component Handler
+
+The emotion component handler make use of this service to increase the performance of loading an shopping world. Imagine you have 3 article boxes configured with topseller products of a category. Prior to 5.3, you will get the same product for each box. As of 5.3, you will get different topseller products.
```
Comment (mitelg) [inline source/developers-guide/shopware-5-search-bundle/index.md]: ... you have three product boxes ...

Comment (janbuecker) [inline source/developers-guide/shopware-5-search-bundle/index.md]: done

File: source/developers-guide/shopware-5-search-bundle/index.md [hunk 2]
```diff
@@ -231,3 +231,48 @@ In most cases, only a list of product numbers is not enough for the store front.
...
+
+## Optimized Batch Search
+
+As of Shopware 5.3, it is possible to search for multiple critera objects at once. In addition, the fetch process for products will be minimized by fetching a product number only once. An optimization service will then try to optimize the search request by combining identical criteria objects into one. So there will be less search requests for identical criterias.
+
+Another advantage is that you won't get the same results with identical searches.
+
+### Example: Emotion Component Handler
+
+The emotion component handler make use of this service to increase the performance of loading an shopping world. Imagine you have three article boxes configured with topseller products of a category. Prior to 5.3, you will get the same product for each box. As of 5.3, you will get different topseller products.
```
Comment (mitelg) [inline source/developers-guide/shopware-5-search-bundle/index.md]: you don't want to write product boxes?

Comment (janbuecker) [inline source/developers-guide/shopware-5-search-bundle/index.md]: Changed 👍

**Comment by mitelg** [comment]
@janbuecker there are conflicts. if you resolved them, you are allowed to merge :smile:

**Pull request merged**